packages/xo-web/src/xo-app/backup/file-restore/index.js

```diff
@@ -86,7 +86,10 @@ export default class Restore extends Component {
   }
 
   _refreshBackupList = async (_remotes = this.props.remotes, jobs = this.props.jobs) => {
-    const remotes = keyBy(filter(_remotes, 'enabled'), 'id')
+    const remotes = keyBy(
+      filter(_remotes, ({ enabled, url }) => enabled && !/^s3\+/.test(url)),
```

Comment:
I think this check should be handled on the server's side, we should had a boolean indicating whether file restore is supported on a given remote.